analyzer/analyzer.go

```diff
@@ -25,25 +26,35 @@ var (
 )
 
 func init() {
-	Analyzer.Flags.StringVar(&flagRules, "rules", "", "path to a gorules file")
+	Analyzer.Flags.StringVar(&flagRules, "rules", "", "comma-separated list of gorule file paths")
 	Analyzer.Flags.StringVar(&flagE, "e", "", "execute a single rule from a given string")
 }
 
+type parseRulesResult struct {
+	rset      *ruleguard.GoRuleSet
+	multiFile bool
+}
+
 func runAnalyzer(pass *analysis.Pass) (interface{}, error) {
 	// TODO(quasilyte): parse config under sync.Once and
 	// create rule sets from it.
 
-	rset, err := readRules()
+	parseResult, err := readRules()
 	if err != nil {
 		return nil, fmt.Errorf("load rules: %v", err)
 	}
+	rset := parseResult.rset
+	multiFile := parseResult.multiFile
 
 	ctx := &ruleguard.Context{
 		Pkg:   pass.Pkg,
 		Types: pass.TypesInfo,
 		Sizes: pass.TypesSizes,
 		Fset:  pass.Fset,
-		Report: func(n ast.Node, msg string, s *ruleguard.Suggestion) {
+		Report: func(info ruleguard.GoRuleInfo, n ast.Node, msg string, s *ruleguard.Suggestion) {
```

Comment:
Hi, I believe this might be a breaking change. For example it breaks: 
https://github.com/go-critic/go-critic/blob/master/checkers/ruleguard_checker.go#L81

not sure what your policy is for breaking changes and semantic versioning, but it will break users of packages relying on golangci, go-critic etc and who use `go get -u` for their dependencies